android/app/src/main/java/com/teamtripdraw/android/ui/common/bindingAdapter/ImageBindingAdapter.kt

```diff
@@ -52,3 +53,19 @@ fun ImageView.setRoundImageWithVisibility(imgUrl: String?) {
             .into(this)
     }
 }
+
+@BindingAdapter("app:setRoundImageWithVisibility")
+fun ImageView.setRoundImageWithVisibility(imgFile: File?) {
+    if (imgFile == null) {
+        this.visibility = View.GONE
+        return
+    } else {
+        this.visibility = View.VISIBLE
+        Glide.with(this.context)
+            .load(imgFile)
+            .placeholder(R.drawable.shape_td_gray_fill_0_rect)
+            .error(R.drawable.shape_td_gray_fill_0_rect)
+            .transform(CenterCrop(), RoundedCorners(20))
+            .into(this)
+    }
```

Comment:
혹시 다음과 같이 early return 해도 무리없이 동작하나요?
```suggestion
    if (imgFile == null) {
        this.visibility = View.GONE
        return
    }
    this.visibility = View.VISIBLE
    Glide.with(this.context)
        .load(imgFile)
        .placeholder(R.drawable.shape_td_gray_fill_0_rect)
        .error(R.drawable.shape_td_gray_fill_0_rect)
        .transform(CenterCrop(), RoundedCorners(20))
        .into(this)
```

Reply:
오! 맞습니당! 둘 다 잘 작동합니다! 
안그래도 이렇게 바꿀까 고민했었는데 다른 메서드와 동일한 방식으로 처리하기 위해 이렇게 두었습니다!
이왕 리뷰 달린거 둘 다 바꿔버려야겠군요 희희